guizero/Waffle.py

```diff
@@ -106,34 +168,22 @@ def get_all(self):
             row = []
             for x in range(self._width):
                 obj = self._save_canvas[y][x]
-                row.append(self._canvas.itemcget(obj,'fill'))
+                row.append(self._canvas.itemcget(obj, "fill"))
             all_pixels.append(row)
         return all_pixels
 
     # Internal use only
     # Detect x,y coords of where the user clicked
-    def _clicked_on(self,e):
+    def _clicked_on(self, e):
         # you can only click on the waffle if its enabled
         if self._enabled:
             canvas = e.widget
             x = canvas.canvasx(e.x)
             y = canvas.canvasy(e.y)
-            pixel_x = int(x/(self._pixel_size+self._pad))
-            pixel_y = int(y/(self._pixel_size+self._pad))
+            pixel_x = int(x / (self._pixel_size + self._pad))
+            pixel_y = int(y / (self._pixel_size + self._pad))
             if self._command:
-                args_expected = utils.no_args_expected(self._command)
-                if args_expected == 0:
```

Comment:
you have removed the code which caters for either 0 or 2 arguments being passed.